### PR TITLE
Integrate JULY-2022 ASB patches to R- stable branch

### PR DESCRIPTION
**Pull request by rrsenapati1**
Tracked-On: OAM-102661
Signed-off-by: Senapati, Rasmiranjan <rasmiranjan.senapati@intel.com>

### CODE REVIEW

**Review by shyjumon-n — APPROVED** [review]
LGTM

File: aosp_diff/preliminary/libcore/02_0002-Reject-non-ASCII-hostnames-and-SANs-.patch [hunk 1]
```diff
@@ -0,0 +1,46 @@
+From be231aa384e6ffb2807af24d8b9d9919394a6c8a Mon Sep 17 00:00:00 2001
```
Comment (tanujtekriwal) [inline aosp_diff/preliminary/libcore/02_0002-Reject-non-ASCII-hostnames-and-SANs-.patch:1]: please remove this patch. Its already there.

**Review by tanujtekriwal — CHANGES_REQUESTED** [review]

**Review by tanujtekriwal — APPROVED** [review]
LGTM

**Review by tanujtekriwal — APPROVED** [review]
LGTM

**Review by shyjumon-n — APPROVED** [review]
Google security patch integration. 
LGTM

**Pull request merged**